sku.0/sys.server/compiled/game/script/item/special/no_trade_removable.java

```diff
@@ -0,0 +1,46 @@
+package script.item.special;
+
+import script.obj_id;
+import script.string_id;
+import script.menu_info;
+import script.menu_info_types;
+
+/**
+ * Allows items that previously had the "noTrade" flag to be made tradeable by
+ * their original owner.
+ * 
+ * Automatically attached to objects in no_trade_removable.tab
+ */
+public class no_trade_removable extends script.base_script {
+    public no_trade_removable() {
+    }
+
+    // Using one of the unused reserved entries.
+    public static final int RADIAL_MENU_ENTRY = menu_info_types.SERVER_MENU47;
```

Comment:
You may want to add a comment to the `menu_info_types` class noting that this reserve option has been used and where/for what. As well as in the companion client and server files that register from this list. 

As a side note, I imagine this change using an existing reserve may cause conflicts with private servers who have made menu additions as this is using an existing reserve and not adding a new one... but that's their problem.

Reply:
Yep! I wanted to add a new one - but given adding a new radial menu item appears to be a breaking change that appears to require a client rebuild, I didn't want to force that for this relatively minor feature.

I've added a comment to the SERVER_MENU I'm using, and the ones I didn't use because they already appeared to be conflicting.